CHANGELOG.md

```diff
@@ -9,13 +9,15 @@ project adheres to [Semantic Versioning](http://semver.org/).
 ==================
 ### Changed
 ### Added
+* BMP support
```

Comment:
I'm guessing it wasn't intentional to move this? ☺️

Reply:
BMP landed after 2.1.0

Reply:
Ah, sorry about that!